applications/web/templates/ingress.yaml

```diff
@@ -1,9 +1,12 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "docker-template.fullname" . -}}
 {{- $svcPort := .Values.service.port -}}
+{{- $allHosts := concat .Values.ingress.hosts, .Values.ingress.porter_hosts -}}
```

Comment:
Uncertain about the syntax (that's why I put the pr in dragit state). Will the comma be an issue?